metron-docker/kafkazk/bin/wait-for-it.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+#   Use this script to test if a given TCP host/port are available
```

Comment:
Something along those lines. Check out the rpm packaging pom.

https://github.com/apache/incubator-metron/blob/master/metron-deployment/packaging/docker/rpm-docker/pom.xml